chain/client/src/info.rs

```diff
@@ -224,6 +225,34 @@ impl InfoHelper {
         self.num_chunks_in_blocks_processed = 0;
         self.gas_used = 0;
 
+        if let Some(telemetry_actor) = &self.telemetry_actor {
```

Comment:
nit: could we also add a comment here saying something like in production, the actor should always be available.  It is just not available in testing.

Reply:
Done